Lucheti29/Training
# News detail frontend implemented

## Summary

'News detail' frontend implemented.
## Test
- Login with a Parse user account (for example, username: t@w.com // password: asdf)
- Change to News tab and click a new to see the detail
- On detail layout, click on 'Back arrow' to go back
## Screenshot

![News detail](http://i19.servimg.com/u/f19/12/28/53/00/news_d11.jpg)
## Cards

https://trello.com/c/3u8rb0Ts/18-news-detail
